### DISCUSSION TITLE
Panel corners

### DISCUSSION THREAD

**Discussion by snipes2** [FedeClaudi/Term.jl#228] [General]
I am running Julia in a Linux Ubuntu in Oracle VMbox on a Windows 11 computer. I am accessing the Linux server with PuTTY set to xterm  When I try the Panel option the vertical and horizontal lines are ok, but where they meet at the corners are not. What do I set/change?

![Corners](https://github.com/FedeClaudi/Term.jl/assets/86324437/da2673a9-66d7-4443-b648-85df0110a8c7)

**Comment by FedeClaudi** [comment]
terminal 🙃 

If the corner characters are not rendered well but the others are fine you can try a different `box` style

**Comment by snipes2** [comment]
That generates 2 questions, how do I determine what styles to choose from? and how do I set the box style? Help at the julia prompt, reading the documentation and searches on Term, don't provide me with answers I can use. I must be too new...

**Comment by FedeClaudi** [comment]
```

julia> print(Panel(; box=:DOUBLE))
╔══════════════════════════════════════════════════════════════════════════════════════╗
║                                                                                      ║
╚══════════════════════════════════════════════════════════════════════════════════════╝
```

you can find the box types [here](https://github.com/FedeClaudi/Term.jl/blob/ae84047dc602247bd74a14e72d91837a75b3f132/src/boxes.jl#L254)

**Comment by snipes2** [comment]
Thank you, that helps this newbee to understand how julia works
![Corners2](https://github.com/FedeClaudi/Term.jl/assets/86324437/1d265617-1fa7-4bc9-bd8f-8d64111b6df2)
